DESCRIPTION

```diff
@@ -35,6 +35,8 @@ Imports:
     utils
 Suggests:
     covr,
+    recipes,
+    workflows,
```

Comment:
Let's keep this list in an alphabetical order.

Reply:
Done.